### ISSUE TITLE
DockerCli.exe -SharedDrives no longer lists shared drives

### ISSUE THREAD

**Issue by NCarlsonMSFT** [docker/for-win#5625]
- [x] I have tried with the latest version of my channel (Stable and Edge)
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior
Running `DockerCli.exe -SharedDrives` outputs a list of drives that have been shared
### Actual behavior
Running `DockerCli.exe -SharedDrives` outputs no text and returns exit code 1
### Information

  - Windows Version: 1909 18363.592
  - Docker Desktop Version: 2.2.0.0 (42247) stable
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM:
Reproduces on physical and virtual machine (Hyper-V)
### Steps to reproduce the behavior
open a command prompt and run:
```
"%ProgramFiles%\Docker\Docker\DockerCli.exe" -SharedDrives
echo %errorlevel%
```
### Impact
This is breaking the error checking for the Docker Tools in Visual Studio 2019:   https://github.com/microsoft/DockerTools/issues/220

**Comment by stephen-turner** [comment]
Thanks for the report, @NCarlsonMSFT. We can confirm that we have reproduced it.

**Comment by ffMathy** [comment]
When will a fix be available?

**Comment by mat007** [comment]
We’ll be doing a new release probably next week.

**Comment by stephen-turner** [comment]
We have a new private build which (we believe) fixes this bug. https://download-stage.docker.com/win/stable/42579/Docker%20Desktop%20Installer.exe. Please try it and let us know how you get on.

**Comment by beliy26** [comment]
> We have a new private build which (we believe) fixes this bug. https://download-stage.docker.com/win/stable/42579/Docker%20Desktop%20Installer.exe. Please try it and let us know how you get on.

This build solved the problem. Only need to restart the PC after installation.

**Comment by NCarlsonMSFT** [comment]
@stephen-turner I ran into some hiccups trying this. But have it running now and it seems to only be a partial fix. It no longer exits with an error code, but it also does not list out the shared drives:
![image](https://user-images.githubusercontent.com/14239288/73772490-1470d880-4735-11ea-8418-dcc89444cd75.png)
![image](https://user-images.githubusercontent.com/14239288/73772587-45510d80-4735-11ea-8e72-381ea3f80b70.png)

**Comment by mat007** [comment]
@NCarlsonMSFT It’s a dash, as in `-SharedDrives`, I think the capitalization also matters.
Any unknown sub-command or flag gets ignored.

Sorry for this but it was meant to be an internal tool only and never received proper UX love…

**Comment by NCarlsonMSFT** [comment]
🤦‍♂️ Thanks, that fixed it.

**Comment by stephen-turner** [comment]
We have just released 2.2.0.3, which fixes this issue, so closing this ticket. In case any Edge users are watching this ticket, it will be on the Edge channel tomorrow too.

**Comment by cjbush** [comment]
Just installed 2.2.0.3 and can now debug containers in VS again. Thanks for the fix!

**Comment by ebriney** [comment]
A fix has been released in Docker Desktop 4.14.0. See release notes for more details.
